src/main/java/pl/gov/coi/pomocua/ads/dictionaries/api/CityLookupResource.java

```diff
@@ -32,11 +34,12 @@ public ResponseEntity<CityLookupResponse> getCities(@RequestParam String query)
             return new ResponseEntity<>(HttpStatus.NO_CONTENT);
         }
 
-        Pageable pageable = PageRequest.of(0, 5, Sort.by(Sort.Direction.ASC, "location.city"));
+        Sort sort = Sort.by(Sort.Direction.ASC, "location.city");
 
         List<CityLookupDto> cities = cityRepository
-                .findByLocationCityWithPageable(query.toLowerCase(), pageable)
+                .findByLocationCityWithSort(query.toLowerCase(), sort)
                 .stream()
+                .limit(cityLookupLimit)
```

Comment:
This limiting should be done on DB-side, as it was done previously, so that we don't fetch too much data over the wires unnecessarily.